stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -736,14 +734,12 @@ function opnormInf(A::AbstractMatrix{T}) where T
     Tnorm = typeof(float(real(zero(T))))
     Tsum = promote_type(Float64, Tnorm)
     nrm::Tsum = 0
-    @inbounds begin
-        for i = 1:m
-            nrmi::Tsum = 0
-            for j = 1:n
-                nrmi += norm(A[i,j])
-            end
-            nrm = max(nrm,nrmi)
+    for i = axes(A,1)
```

Comment:
Do you need the line `m, n = size(A)` anymore? Looks like dead code, so we could just as well remove it. Also, personally I prefer `for i in axes(A, 1)` over the matlabian `i =`, but that's just a preference.